mkt/websites/serializers.py

```diff
@@ -10,19 +10,18 @@
 class WebsiteSerializer(serializers.ModelSerializer):
     categories = ListField(serializers.CharField())
     description = TranslationSerializerField()
-    device_types = ListField(serializers.CharField(), source='device_names')
     id = serializers.IntegerField(source='pk')
     short_name = TranslationSerializerField()
     name = TranslationSerializerField()
     title = TranslationSerializerField()
     icons = serializers.SerializerMethodField('get_icons')
 
-    # FIXME: keywords, regions. try to stay compatible with Webapp API
-    # as much as possible.
+    # FIXME: keywords. need to refactor Webapp code. Probably want to use
+    # tag_list and associated transformer to keep things DRY and coherent.
 
     class Meta:
         model = Website
-        fields = ['categories', 'description', 'device_types', 'icons', 'id',
```

Comment:
Not exposing `device_types` any longer in the API for websites since they are marked as compatible with all devices anyway.